data/json/mapgen/robofaq_locs/robofac_hq_chunks.json

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "mapgen",
+  "method": "json",
+  "nested_mapgen_id": "robofac_hq_surface_freemerchant_camp",
+  "object": {
+    "faction_owner": [ { "id": "free_merchants", "x": [ 0, 14 ], "y": [ 0, 14 ] } ],
+    "mapgensize": [ 15, 15 ],
+    "rows": [
+      "        ...... ",
+      " ###### ...... ",
+      " #x.rrrr.......",
+      " #t.rrrr.......",
+      " #x.rrrr...F...",
+      " #+++##..h.....",
+      " #c.t#v........",
+      " #####v........",
+      "       #+# #+# ",
+      "       #c# #c# ",
+      "       ### ### ",
+      "               ",
+      "               ",
+      "               ",
+      "               "
+    ],
+    "terrain": {
+      ".": "t_dirt",
+      "r": "t_carpet_yellow",
+      "M": "t_pavement",
+      "#": "t_dirt",
+      "x": "t_dirt",
+      "c": "t_dirt",
+      "F": "t_pit_shallow"
+    },
+    "furniture": { "#": "f_skin_wall", "+": "f_skin_door", "x": "f_table", "t": "f_table", "h": "f_chair", "F": "f_firering" },
+    "traps": { "c": "tr_fur_rollmat", "v": "tr_funnel" },
+    "place_npcs": [
+      { "class": "roabofac_free_merchant", "x": 5, "y": 3 },
+      { "class": "guard", "x": 7, "y": 7 },
+      { "class": "guard", "x": 10, "y": 4 }
+    ],
+    "items": { "c": { "item": "bed", "chance": 100 } },
+    "item": {
+      "r": { "item": "pillow", "chance": 33 },
+      "F": { "item": "pot", "chance": 100 },
+      "v": { "item": "jug_plastic", "chance": 100 }
+    },
+    "place_items": [ { "item": "robofac_basic_trade", "chance": 94, "x": 2, "y": [ 2, 4 ] } ]
```

Comment:
No matter how I tell the game to spawn these items, they never spawn I have tried "items", "loot" and "place_items"

Im not sure whats causing it, Maybe its some load order related thing?